### PR TITLE
Add `download` attribute for `<a>` on `<video>` element page's examples

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

inspired by #29287

### CODE REVIEW

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @skyclouds2001, thanks for adding the `download` attribute! The changes look good, except for one case where I've left a question.

File: files/en-us/web/html/element/video/index.md [hunk 1]
```diff
@@ -373,7 +373,11 @@ This example plays a video when activated, providing the user with the browser's
   poster="https://peach.blender.org/wp-content/uploads/title_anouncement.jpg?x11217"
   width="620">
   Sorry, your browser doesn't support embedded videos, but don't worry, you can
-  <a href="https://archive.org/details/BigBuckBunny_124">download it</a>
+  <a
+    href="https://archive.org/details/BigBuckBunny_124"
+    download="BigBuckBunny_124"
+    >download it</a
+  >
```
Comment (skyclouds2001) [inline files/en-us/web/html/element/video/index.md:380]:
```suggestion
  <a href="https://archive.org/details/BigBuckBunny_124">download it</a>
```

File: files/en-us/web/html/element/video/index.md [hunk 2]
```diff
@@ -373,7 +373,11 @@ This example plays a video when activated, providing the user with the browser's
   poster="https://peach.blender.org/wp-content/uploads/title_anouncement.jpg?x11217"
   width="620">
   Sorry, your browser doesn't support embedded videos, but don't worry, you can
-  <a href="https://archive.org/details/BigBuckBunny_124">download it</a>
+  <a
+    href="https://archive.org/details/BigBuckBunny_124"
+    download="BigBuckBunny_124"
```
[the remark this answers is not in the document]
Comment (skyclouds2001) [reply files/en-us/web/html/element/video/index.md]: yes, the link is a website, so the `download` attribute should not be added

**Review by dipikabh — APPROVED** [review]
Thanks for your contribution, @skyclouds2001!

**Pull request merged**